src/tools/components/MSAWrapper.tsx

```diff
@@ -83,33 +80,24 @@ const MSAWrapper: React.FC<{
         }
       : {};
 
+  const handleHighlightSelect = useCallback(
+    ({ id }: { id: MsaColorScheme }) => setHighlightProperty(id),
+    []
+  );
+
   return (
     <>
       <div className="button-group">
-        <DropdownButton label="Highlight properties" className="tertiary">
-          <div className="dropdown-menu__content">
-            <ul>
-              {Object.entries(msaColorSchemeToString).map(
-                ([schemeValue, schemeString]) => (
-                  <li key={schemeString}>
-                    <button
-                      type="button"
-                      className={cn('button', {
-                        primary: highlightProperty === schemeValue,
-                        tertiary: highlightProperty !== schemeValue,
-                      })}
-                      onClick={() =>
-                        setHighlightProperty(schemeValue as MsaColorScheme)
-                      }
-                    >
-                      {schemeString}
-                    </button>
-                  </li>
-                )
-              )}
-            </ul>
-          </div>
-        </DropdownButton>
+        <TreeSelect
+          data={colorSchemeTree}
+          onSelect={handleHighlightSelect}
+          autocomplete
+          autocompleteFilter
+          autocompletePlaceholder="Enter a color scheme"
+          label="Highlight properties"
```

Comment:
How it is now is perfectly fine but I am wondering if it would be useful to change the label to the selected colour scheme?